crmd/control.c

```diff
@@ -952,6 +952,9 @@ pe_cluster_option crmd_opts[] = {
 	{ "stonith-watchdog-timeout", NULL, "time", NULL, NULL, &check_sbd_timeout,
 	  "How long to wait before we can assume nodes are safely down", NULL
         },
+        { "stonith-max-attempts",NULL,"integer",NULL,"10",&check_positive_number,
+          "The maximum number of stonith attempts before giving up."
```

Comment:
Let's reword this as:

    How many times stonith can fail before it will no longer be attempted on a target

It's not much different, but it highlights that we will give up only for the target that failed, not all targets.